docs/DeveloperGuide.adoc

```diff
@@ -422,46 +422,75 @@ image::TimeTableSequenceDiagram.png[width="800"]
 {nbsp}
 
 // tag::import_export_timetable[]
-=== Import timetable and Export timetable feature
+=== Import and Export timetable feature
+
+The import and export timetable feature allows users to import and export their timetables from/to external applications as .ics files.
+
+The currently supported applications are: +
+
+Import:
+
+* NUSMODS [https://nusmods.com] +
+
+Export:
+
+* FreeTime (you export and then import it back in)
+
+* Google Calendar (Preliminary support: TimeSlots show up as individual events for 1 week.)
+
 ==== Current implementation
 
-The import timetable feature allows users to import their timetables from NUSMODS, as an .ics file.
+The import and export feature consists of two parts: +
 
-The timetable feature consists of two parts: +
-1. The backend, which handles the reading of the .ics file and the creation of the timetable object. +
+1. The backend, which handles the reading and writing of _.ics_ files, and the conversion between the .ics format and the TimeTable object. +
```

Comment:
Looks like you're missing some formatting on the second `.ics`

Reply:
aye